n2ws/manifest.json

```diff
@@ -24,9 +24,9 @@
   "assets": {
     "configuration": {},
     "dashboards": {
-	  "N2WSBackup&Recovery-Graphicalversion": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion.json",
-	  "N2WSBackup&Recovery-Graphicalversion-areas": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json",
-	  "N2WSBackup&Recovery-Squaresdashboard": "assets/dashboards/N2WSBackup&Recovery-Squaresdashboard.json"
+	  "N2WSBackup&Recovery-EntitiesSpecificDashboard": "assets/dashboards/N2WSBackup&Recovery-EntitiesSpecificDashboard.json",
+	  "N2WSBackup&Recovery-GraphicalVersion": "assets/dashboards/N2WSBackup&Recovery-GraphicalVersion.json",
+	  "N2WSBackup&Recovery-GraphicalVersion-Areas": "assets/dashboards/N2WSBackup&Recovery-GraphicalVersion-Areas.json"
```

Comment:
Dahsboard validation is failing with following errors:
```
Validating all Dashboard definition files...
n2ws...  FAILED
  assets/dashboards/N2WSBackup&Recovery-GraphicalVersion.json does not exist
n2ws...  FAILED
  assets/dashboards/N2WSBackup&Recovery-GraphicalVersion-Areas.json does not exist
15 valid files
2 invalid files
```
You can run this validation locally with `ddev validate dashboards`

Reply:
the file assets/dashboards/N2WSBackup&Recovery-GraphicalVersion-Areas.json does not exist, instead the n2ws/assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json does. Note that in one areas has caps A and in the other one is lowercase.
The same happens with GraphicalVersion.

Reply:
Hi, I tried to run `ddev validate dashboards` without adding "n2ws" to the path and it passed, failed with "n2ws".
In addition, I committed with "n2ws" and it failed. So the checks fails either way ...

Reply:
If you're running this on windows or macos, that have case insensitive file systems the validation will pass